docs/persistence/elastic-search.md

```diff
@@ -0,0 +1,20 @@
+# Elastic Search
+
+!!! Info
+    This feature is only available in GCP clusters.
```

Comment:
maybe point readers straight to the document? e.g. `[GCP clusters](../clusters/gcp/README.md)` here?

Reply:
Sure, kopierte den fra en annen side for å ha de litt like.